### PR TITLE
Usagov 1878 clear announce

### PR DESCRIPTION
**Pull request by omerida**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1878

## Description

Announce that selections were cleared when using a screen reader.

## Type of Changes

- [ ] New Feature
- [x] Bugfix
- [ ] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like
- [ ] Drupal Config (requires "drush cim")
- [ ] New Modules (requires rebuild)
- [ ] Documentation
- [ ] Infrastructure
  - [ ] CMS
  - [ ] WAF
  - [ ] Egress
  - [ ] Tools
- [ ] Other

## Testing Instructions

Need a screen reader like NVDA active while using the benefits search page.

### Change Requirements

- [ ] Requires New Documentation (Link: {})
- [ ] Requires New Config
- [ ] Requires New Content

### Validation Steps

- There should be no visual changes to how the search works.
- Screen reader should announce that your selections were cleared when you uncheck the "Select All" button (check once to toggle all on, then again to uncheck all)
- Screen reader should announce that your selections were cleared when you press the "Clear Selectoins" button IF you have one or more categories checked.
- Should also read the Error message if you submit selections with none checked.

## Security Review

- [ ] Adds/updates software (including a library or Drupal module)
- [ ] Communication with external service
- [ ] Changes permissions or workflow
- [ ] Requires SSPP updates

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

### CODE REVIEW

File: web/modules/custom/usagov_benefit_category_search/js/benefitSearch.js [hunk 1]
```diff
@@ -641,7 +640,7 @@ jQuery(document).ready(async function () {
     lifeEventsPath = "../../_data/benefits-search/es/life-events.json";
     labels = {
       'showingResults': '@first@&ndash;@last@ de @totalItems@ resultados',
-      'page': "página",
+      'page': "page",
```
Comment (akf) [inline web/modules/custom/usagov_benefit_category_search/js/benefitSearch.js]: It appears you're not actually using the "page" field, should it just go away? (Just commenting for now; I'll review this later when I'm not running tome on another branch.)

File: web/themes/custom/usagov/scripts/pagination.js [hunk 2]
```diff
@@ -142,7 +142,7 @@ function Pagination(total, current, assetBase, labels, onClick) {
     if (num === this.current) {
       // highlight this page if it's the current one
       atag.classList.add('usa-current');
-      atag.setAttribute('aria-current', myself.labels.page);
+      atag.setAttribute('aria-current', 'page');
```
Comment (omerida) [inline web/themes/custom/usagov/scripts/pagination.js:145]: aria-current has a limited set of allowed values

File: web/modules/custom/usagov_benefit_category_search/js/benefitSearch.js (hunk 1 above)
Comment (omerida) [reply web/modules/custom/usagov_benefit_category_search/js/benefitSearch.js]: This was a mistake, these labels are used in the pagination. I was trying to fix another accessibility issue that siteimprove's browser plugin raised. The aria-current attribute isn't supposed to be translated. must be just 'page'

**Review by akf — APPROVED** [review]
I don't have NVDA, but I was able to test with VoiceOver. VO is finicky; whether I get these messages seems to depend on how I select the controls, but if I do it in the particular way VO suggests I do hear the announcements. 

It looks the same as it always did!

**Pull request merged**